src/components/gutter-control/gutter-wrapper.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { cloneElement, isValidElement } from '@wordpress/element';
+
+/**
+ * Return an element Wrapped with Gutter Properties.
+ *
+ * @return {JSX.Element}  GutterWrapper component
+ */
+function GutterWrapper( { children, gutter, gutterCustom, className, condition = true } ) {
+	if ( ! isValidElement( children ) ) {
+		return children;
+	}
+
+	let style = {};
+
+	if ( children.props.style ) {
+		style = {
+			...children.props.style,
+		};
+	}
+
+	if ( 'custom' === gutter ) {
```

Comment:
The `condition` prop should also determine whether we apply the style prop to the element like it decides the addition of our className below.

Reply:
@jrtashjian are you talking about not having empty style attribute?

Reply:
yeah, we don't want to render an attribute we're not using.

Reply:
Fixed in 7e66cfb